net-tools/netwatch/src/netmon/actor.rs

```diff
@@ -127,10 +126,14 @@ impl Actor {
                         debounce_interval.reset_immediately();
                     }
                 }
-                Some(_event) = self.mon_receiver.recv() => {
-                    trace!("network activity detected");
-                    last_event.replace(false);
-                    debounce_interval.reset_immediately();
+                Some(event) = self.mon_receiver.recv() => {
+                    match event {
+                        NetworkMessage::Change => {
```

Comment:
`Some(NetworkMessage::Change) = self.mon_receiver.recv()` might be a nicer way to write this?

I also noticed that `NetworkMessage::Change` is marked with `#[allow(dead_code)]`.  This seems suspicious, can we remove that marker as well?

Reply:
> This seems suspicious, can we remove that marker as well?

no because this message is not used on android, and I didn't want to start adding cfgs to lint hints

Reply:
> `Some(NetworkMessage::Change) = self.mon_receiver.recv()` might be a nicer way to write this?

maybe, we usually always put the match inside, so I wrote it like that for consistency

Reply:
see issues with tokio::select, I cleaned this up to not use an `else`branch